ui/plugins/auto_classification/controller.js

```diff
@@ -839,10 +839,12 @@ treeherder.controller('ClassificationPluginCtrl', [
 
             // If we have a TextLogSummaryModel then we should have enough data to
             // load this panel
-            var checkLoaded = ThTextLogSummaryModel.get($scope.jobId,
-                                                        {timeout: requestPromise,
-                                                         cache: true});
-            checkLoaded
+            ThTextLogSummaryModel
+                .get($scope.jobId,
+                     {timeout: requestPromise,
+                      cache: true})
+                .then(() => true)
+                .catch(() => false)
                 .then(function(loaded) {
```

Comment:
Well, no. In this case there are background tasks that have to complete before the data is available, so it makes sense to keep trying to reload until the data is there.